src/System.Resources.ReaderWriter/tests/ResourceReaderUnitTest.cs

```diff
@@ -40,35 +33,35 @@ public static ResourceWriter GenerateResourceStream(Dictionary<string, string> i
             return rw;
         }
 
+
         [Fact]
-        public static void ReadResource()
+        public static void ReadResourceUsingTypedEnumerator()
         {
-
-            using (var ms2 = new MemoryStream())
+            using (var stream = new MemoryStream())
             {
-                using (var rw = GenerateResourceStream(s_dict, ms2))
+                using (var resourceWriter = GenerateResourceStream(s_dict, stream))
                 {
                     //Rewind to begining of stream
 
-                    ms2.Seek(0L, SeekOrigin.Begin);
+                    stream.Seek(0L, SeekOrigin.Begin);
```

Comment:
There are a bunch of changes here that are stylistic and unrelated to the primary purpose of the change, that of changing the Enumerator to be a struct.  In the future, it'd be good to keep such changes in a separate commit.  Otherwise someone looking through history is going to see all of these changes as being related to that Enumerator change.